spec/suma/i18n_spec.rb

```diff
@@ -105,6 +105,21 @@
       J
     end
 
+    it "ensures interpolation values are unchanged and removes whitespaces" do
+      path = described_class.strings_path("es")
+      File.write(path, '{"x":1}') # Make sure it gets blown away
+      csv = "Key,Spanish,English\n" \
+            "food:price_times_quantity,{{ precio }} x {{ cantidad }},{{price}} x {{quantity}}"
```

Comment:
Use a different value than 'x' in each string- right now it is unclear whether you're even reading the english vs. spanish string, for example (ie, use `{{ precio }} es {{cantidad}}` and `{{price}} en {{ quanity }}`)